config/routes.rb

```diff
@@ -7,10 +7,14 @@
 
     get "/signup", to: "users#new"
     post "/signup", to: "users#create"
-    resources :users, only: %i(new create show)
+    resources :users, only: %i(show new create destroy edit update)
```

Comment:
comprehensive actions
```suggestion
    resources :users
```